components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -31,6 +31,9 @@
 from strenum import KebabCaseStrEnum
 
 # CONSTANTS
+DECOMPRESSION_COMMAND = "x"
```

Comment:
How about:

* `EXTRACT_FILE_CMD` and `EXTRACT_IR_CMD`
* `handle_extract_file_cmd` and `handle_extract_ir_cmd`